manager/config/prod/deployment_configmap.yaml

```diff
@@ -9,24 +9,31 @@ data:
   SECRET_PROVIDER_URL: "http://secret-provider.m4d-system.svc.cluster.local:5555/get-secret"
   SECRET_PROVIDER_ROLE: "demo"
   #catalog connector
-  CATALOG_CONNECTOR_URL: "egr-connector:50084"
-  CATALOG_PROVIDER_NAME: "EGERIA"
+  # CATALOG_CONNECTOR_URL: "egr-connector:50084"
+  # CATALOG_PROVIDER_NAME: "EGERIA"
+  CATALOG_CONNECTOR_URL: "wkc-connector:50090"
+  CATALOG_PROVIDER_NAME: "WKC"
 
   #credentials manager connector
-  CREDENTIALS_CONNECTOR_URL: "vault-connector:50083"
-  CREDENTIALS_PROVIDER_NAME: "VAULT"
+  #CREDENTIALS_CONNECTOR_URL: "vault-connector:50083"
+  #CREDENTIALS_PROVIDER_NAME: "VAULT"
+  CREDENTIALS_CONNECTOR_URL: "wkc-connector:50090"
+  CREDENTIALS_PROVIDER_NAME: "WKC"
 
   #policy-compiler and policy manager connectors
-  MAIN_POLICY_MANAGER_CONNECTOR_URL: "opa-connector:50082"
-  MAIN_POLICY_MANAGER_NAME: "OPA"
+  # MAIN_POLICY_MANAGER_CONNECTOR_URL: "opa-connector:50082"
+  # MAIN_POLICY_MANAGER_NAME: "OPA"
+  # USE_EXTENSIONPOLICY_MANAGER: "false"
+  MAIN_POLICY_MANAGER_CONNECTOR_URL: "wkc-connector:50090"
+  MAIN_POLICY_MANAGER_NAME: "WKC"
   USE_EXTENSIONPOLICY_MANAGER: "false"
   #EXTENSIONS_POLICY_MANAGER_CONNECTOR_URL: "opa-connector:50082"
   #EXTENSIONS_POLICY_MANAGER_NAME: "OPA"
 
   #internal vault related variables
   VAULT_ADDRESS: "http://vault.m4d-system.svc.cluster.local:8200/"
   VAULT_DATASET_HOME: "m4d/dataset-creds/"
-  VAULT_USER_HOME: "m4d/user-creds/"
+  VAULT_USER_HOME: "secret/m4d/user-creds/"
```

Comment:
why?

Reply:
when I tried to add to the m4d path it was not working as I think the m4d path is not enabled by default(?) when I last checked. The 'secret' path is enabled by default and hence I used that. Anyways I will check again and confirm on this point. Thanks.